server/routes/creatingReports/addInvolvedStaff.test.ts

```diff
@@ -1,9 +1,9 @@
 import request from 'supertest'
-import { StaffDetails } from '../data/draftReportClientTypes'
-import DraftReportService, { AddStaffResult } from '../services/report/draftReportService'
-import { appWithAllRoutes, user } from './testutils/appSetup'
+import { StaffDetails } from '../../data/draftReportClientTypes'
+import DraftReportService, { AddStaffResult } from '../../services/drafts/draftReportService'
+import { appWithAllRoutes, user } from '../__test/appSetup'
```

Comment:
test resources are along side normal code - making this more apparent by adopting this naming convention for the folder